applications/nrf_desktop/src/modules/Kconfig.hid

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "HID"
+
+config DESKTOP_HID_REPORT_DESC
+	string "HID report descriptor"
+	default ""
+	help
+	  This option specifies a path to the HID report descriptor file.
+
+config DESKTOP_HID_MOUSE
```

Comment:
Maybe we could add this as select (to ensure that device will have only one type) ?

Reply:
I prefer to treat it as a shortcut to enable minimum set of reports for a given device. User can manually enable additional reports anyway.